mail/postfix/src/opnsense/mvc/app/models/OPNsense/Postfix/General.xml

```diff
@@ -185,5 +185,12 @@
             <default>0</default>
             <Required>Y</Required>
         </reject_unverified_recipient>
+        <delay_warning_time type="IntegerField">
+            <default>0</default>
+            <Required>N</Required>
```

Comment:
With your templating required should be yes, otherwise you can leave it empty and config is wrong

Reply:
Doesn't the system fill in '0' if the user leaves the field empty? What else does the default setting do if not that?

Reply:
Default is 0 but with required no you can remove the 0 and value of it doesnt match !='0'

Reply:
OK I just tried it. When I empty the field and save, it stays empty. But the config line does not end up in the main.cf - correctly so - because the conditions in the template seem to work as intended.

So I could change the allowed range to 1 - 24 and the help to "leave empty to disable" and all is well? What do you think? Another checkbox to tick and specifically enable the feature followed by a mandatory field just clutters the interface, IMHO.

Reply:
If you leave the field empty I would expect the delay warning time just to be "h", but I'm not at a computer now.

Reply:
@mimugmail @pmhausen hi! with empty general.delay_warning_time template just skips it (delay_warning_time is realy empty in conf, not null). looks like `if helpers.exists` does its job

Reply:
"empty or 0" may be?)

Reply:
Fine with me.

Reply:
I need some help now with github. So if I update and push to my forked repo, how do I update the pull request? Generate a new one instead?

Thanks!

Reply:
Thanks for the help, folks.